[wip]: Reviewing-mmaler - :Add Network Troubleshooting section (#1438)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_websocket-troubleshooting.adoc

// Module included in the following assemblies:
//
// websocket-troubleshooting

[id="websocket-troubleshooting_{context}"]

> = WebSocket Troubleshooting

Reviewer: ```suggestion
= Troubleshooting WebSocket connections
```

Reviewer: This procedure seems connected to the previous one ("Network requirement"). Perhaps the preceding topic could be made more concise and incorporated into this procedure as prerequisites?

> If a WebSocket Secure (WSS) connection fails, the following instructions can help to determine the reason and find the solution.

Reviewer: ```suggestion
To troubleshoot a failing WebSocket Secure (WSS) connection, use the instructions in this section.
```

.Prerequisites

Using an unsupported web browser causes a connection interruption, followed by a warning message.

* Use a supported Web browser:

** Chrome
** Firefox

.Procedure

. Browser support
.. Check if WebSockets is running by a link:https://websocketstest.com/[realtime web test] using one of the supported browsers.
		
		
. Proxy servers and firewalls settings
.. Ask a system administrator to check if there is a proxy server or firewall that blocks WebSocket Secure (WSS) connections on port 443. 
+
* Possible required actions:

** Add an exception to the firewall 
** Have the proxy intercept WebSocket connection 

> .. Check if WebSockets is running by a link:https://websocketstest.com/[realtime web test] using one of the supported browsers.

Reviewer: Why is this step duplicated?

Author: This step is duplicate as a websocket health check
